src/hotspot/cpu/x86/sharedRuntime_x86.cpp

```diff
@@ -83,3 +84,35 @@ void SharedRuntime::inline_check_hashcode_from_object_header(MacroAssembler* mas
 }
 #endif //COMPILER1
 
+JRT_LEAF(jfloat, SharedRuntime::frem(jfloat  x, jfloat  y))
+// see SharedRuntime::drem
+  jfloat retval;
+  asm ("\
+1:               \n\
+fprem            \n\
+fnstsw %%ax      \n\
+test   $0x4,%%ah \n\
+jne    1b        \n\
+"
+    :"=t"(retval)
```

Comment:
This doesn't compile on Windows
```
 c:\sb\prod\1677461706\workspace\open\src\hotspot\cpu\x86\sharedRuntime_x86.cpp(97): error C2059: syntax error: ':'
c:\sb\prod\1677461706\workspace\open\src\hotspot\cpu\x86\sharedRuntime_x86.cpp(114): error C2059: syntax error: ':'
```
I assume this asm is gcc based.

Reply:
Thanks for reporting it, I am no longer providing the asm variant on any Windows platform.